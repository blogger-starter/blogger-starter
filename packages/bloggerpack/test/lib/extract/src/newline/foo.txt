<extract-start>



foo



</extract-end>
