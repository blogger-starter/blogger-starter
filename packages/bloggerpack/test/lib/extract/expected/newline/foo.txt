/* src/newline/foo.txt */

foo
